pkg/libvmi/vmi.go

```diff
@@ -269,6 +271,24 @@ func WithoutSerialConsole() Option {
 	}
 }
 
+func WithActivePods(pods map[types.UID]string) Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		vmi.Status.ActivePods = pods
+	}
+}
```

Comment:
We have `libvmistatus` now, so this is duplicated.
The libvmistatus can be used with:
`vmi := libvmi.New(libvmistatus.WithStatus(libvmistatus.New(libvmistatus.WithActivePod("uid", "nodename"))))